earth_enterprise/src/server/wsgi/common/utils.py

```diff
@@ -23,9 +23,12 @@
 import re
 import socket
 import urllib2
-import xml.sax.saxutils as saxutils
+import 
+import defusedxml.sax as DefusedSax
+from xml.sax import saxutils
 from geAbstractionFetcher import GetHostName
 
+DefusedSax.saxutils = saxutils
```

Comment:
Is the purpose of this to prevent incorrect error reporting by Bandit?

Reply:
This is because defusedxml.sax doesn't contain saxutils. This was an issue reported by Bandit though.

Reply:
Yeah, my wrapper suggestion doesn't make as much sense here. It's most helpful when we're using some functions from defusedxml and some from the regular xml package. Here it might be simpler to just leave the old import.

If you want to get fancy, you could have a completely separate .py file that creates a package that combines the pieces we need from defusedxml and python xml and just have a single import statement here (and everywhere else we use xml) that imports that package.